src/components/DefaultStyleListDialog.vue

```diff
@@ -163,17 +163,30 @@ watch([() => props.modelValue], async ([newValue]) => {
       .filter(
         (characterInfo) => characterInfo.metas.styles.length > 1
       ) as CharacterInfo[];
-    selectedStyleIndexes.value = Object.fromEntries([
-      ...store.state.userCharacterOrder.map((speakerUuid) => [speakerUuid, 0]),
-      ...store.state.defaultStyleIds.map((defaultStyle) => [
-        defaultStyle.speakerUuid,
-        characterInfosMap.value[
-          defaultStyle.speakerUuid
-        ]?.metas.styles.findIndex(
-          (style) => style.styleId === defaultStyle.defaultStyleId
-        ) ?? 0,
-      ]),
-    ]);
+    selectedStyleIndexes.value = Object.fromEntries(
```

Comment:
ここにあると良いコメントとしては、「エンジン未起動状態でデフォルトスタイル選択ダイアログを開くと、未起動エンジンのキャラのデフォルトスタイルが消えてしまう」ことを書くことかなと思いました！
なので、こう･･･？
```suggestion
    // FIXME: エンジン未起動状態でデフォルトスタイル選択ダイアログを開くと
    // 未起動エンジンのキャラのデフォルトスタイルが消えてしまう
    selectedStyleIndexes.value = Object.fromEntries(
```